apps/src/templates/studioHomepages/Courses.jsx

```diff
@@ -81,7 +81,7 @@ const Courses = React.createClass({
           />
         )}
 
-        {courses && (
+        {courses && courses.length > 0 && (
```

Comment:
it did seem that checking for courses seemed redundant, but now agreed that it is necessary because there are times when the user is signed out on /courses and courses aren't passed to this component so course.length throws an error back that there aren't any courses to check the length of